### PR TITLE
Add 2 scenarios

### PR DESCRIPTION
**Pull request by jtart**
JIRA: https://jira.dev.bbc.co.uk/browse/NEWSART-545

_Adds 2 scenario variants as JSON examples in the `/data` directory._

* [x] JIRA ticket added
* [x] Pull request URL added to JIRA ticket
* [x] Up-to-date with latest - `git pull --rebase origin latest`
* [x] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)
* [ ] ~~Tests added for new features~~
* [x] Tests pass - `npm test`
* [x] E2E tests pass - `npm run test:e2e` (requires server running see [README](https://github.com/bbc/simorgh#tests) for details)

* [ ] Test engineer approval

### CODE REVIEW

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
JSON is linted correctly. I will mostly leave checking the data is correct to QA. But there is one problem in the blockId - these are meant to be unique for a block globally, but block 2 in each example are different.

**Review by jamesbhobbs — DISMISSED** [review]
The branch name is confusing but the code change is fine

File: data/scenario-17.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{
+  "locatorId": "c000000000ho",
+  "passport": {
+    "language": "en-gb",
+    "home": "http://www.bbc.co.uk/ontologies/passport/home/News",
+    "articleType": "analysis",
+    "genre": null
+  },
+  "publishedDateTime": "2018-06-04T00:00:00.000Z",
```
Comment (pjlee11) [inline data/scenario-17.json]: Probably best not to have this at midnight for BST/GMT conversion, we can handle the BST logic in a more unique fixture.

**Review by jamesbhobbs — APPROVED** [review]

File: data/scenario-17.json [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      "externalId": "p060njgk",
+      "duration": "PT1M31S",
+      "allowOffsiteEmbedding": false
+    },
+    {
+      "blockId": "0998da6b-51bb-4a0a-8430-324d07b5a325",
+      "type": "image",
+      "aspectRatio": "16:9",
+      "relPath":
+        "/cpsprodpb/14489/production/_101218038_91c17269-57b8-44dc-b123-64fb93bde785.jpg",
```
Comment (pjlee11) [inline data/scenario-17.json]: Will this be an ichef URL from Ares, one to ask investigate

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Pull request merged**